docs/guides/debugging.md

```diff
@@ -149,6 +149,30 @@ places. You can always load it by hand if GDB refuses or fails to load it:
 
     (gdb) source /path/to/your/.gdbinit
 
+Here's an example of a `.gdbinit` file helpful for debugging scylla.
+Note that it registers pretty-printer helpers for printing common
+std-c++ stl containers:
+```
+handle SIG34 pass noprint
+handle SIGUSR1 pass noprint
+set print pretty
+set print static-members no
```

Comment:
This is only needed if you are experiencing crashes. Not seen on more recent gdb versions.

Reply:
will remove